### PR TITLE
Change Register Pair from Gov Proposal to Transaction

### PR DESCRIPTION
**Pull request by Kbhat1**
## Describe your changes and provide context
- Add a transaction MsgRegisterPairs to register a pair for a contract
- Remove gov proposals for registering pairs
## Testing performed to validate your change
- Tested e2e tx for registering pair and queried the registered pair to verify 
- Added unit tests

### CODE REVIEW

File: x/dex/keeper/msgserver/msg_server_register_pairs.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/dex/types"
+)
+
+func (k msgServer) RegisterPairs(goCtx context.Context, msg *types.MsgRegisterPairs) (*types.MsgRegisterPairsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	// TODO: add validation such that only the user who stored the code can register pairs
```
Comment (LCyson) [inline x/dex/keeper/msgserver/msg_server_register_pairs.go]: can we add basic msg validity check (like pair != empty, creator exists and etc), and unit tests for those checks

File: x/dex/types/message_register_pairs.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	}
+	return []sdk.AccAddress{creator}
+}
+
+func (msg *MsgRegisterPairs) GetSignBytes() []byte {
+	bz := ModuleCdc.MustMarshalJSON(msg)
+	return sdk.MustSortJSON(bz)
+}
+
+func (msg *MsgRegisterPairs) ValidateBasic() error {
```
Comment (LCyson) [inline x/dex/types/message_register_pairs.go:45]: can just add msg check here

**Review by LCyson — APPROVED** [review]

File: x/dex/keeper/msgserver/msg_server_register_pairs.go (hunk 1 above)
Comment (Kbhat1) [reply x/dex/keeper/msgserver/msg_server_register_pairs.go]: Added as part of `ValidateBasic`

File: x/dex/types/message_register_pairs.go (hunk 2 above)
Comment (Kbhat1) [reply x/dex/types/message_register_pairs.go:45]: added on latest commit

**Pull request merged**